contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -472,9 +472,20 @@ namespace eosiosystem {
    };
 
 
+   struct [[eosio::table, eosio::contract("eosio.system")]] gifted_ram {
+      name      giftee;
+      name      gifter;
+      int64_t   ram_bytes = 0;
+
+      bool      is_empty()    const { return ram_bytes == 0; }
```

Comment:
The `is_empty()` function does not appear to be necessary.

Reply:
https://github.com/eosnetworkfoundation/eos-system-contracts/pull/168/commits/d364e2732bef7d7cceb2148a3dd8cc2ca3b07d84